tracer/test/test-applications/integrations/Samples.Console/Program.cs

```diff
@@ -117,6 +117,8 @@ private static async Task AsyncMain(string[] args)
                 {
                     Console.WriteLine("Ready");
                     Console.WriteLine($"Echo: {Console.ReadLine()}");
+                    // Small delay to ensure the test has time to read the output before the process exits
+                    Thread.Sleep(100);
```

Comment:
nit: I don't think we should add this, it just slows down dozens of tests without any real benefit IMO